STANDARD.md

```diff
@@ -584,19 +590,37 @@ patch order:
 Unit Systems and Dimensionality
 -------------------------------
 
-While this standard does not impose any unit system on the data that is stored
-itself, it still requires a common interface to convert one unit system to
-another.
-
-To allow scaling data without reformatting it during the write process we
-provide a unit conversation factor, often called `unitSI` in the document,
-to transform it to a corresponding quantity in the International System of
-Units (SI).
+All datasets and attributes can be written in arbitrary units, in the openPMD
```

Comment:
superfluous `,`?

Reply:
Also I would avoid "the openPMD file" in new text. openPMD is not bound to files. Just write `in openPMD` or skip it at all - openPMD is what these docs are about :)